test/fw/bin/pbs_snapshot

```diff
@@ -283,6 +287,14 @@ if __name__ == '__main__':
         usage()
         sys.exit(1)
 
+    if not basic:
+        # Capture 5 days of daemon logs and 30 days of accounting logs
+        # by default
+        if daemon_logs == 0:
+            daemon_logs = 5
+        if acct_logs == 0:
+            acct_logs = 30
```

Comment:
So, we don't wanna capture any logs by default when --basic is True, but --basic can be paired with --accounting-logs/--daemon-logs options to explicitly ask pbs_snapshot to also capture some logs with --basic. So, I can no longer initialize them to their default values and then set them to 0 inside/after the getopt loop because I won't know whether the user wants only basic or basic + some logs. That's why I need to initialize them to 0 and set them to their default values here if --basic hasn't been provided. I hope I'm making sense.

Actually, I need to initialize them to -1 or None, otherwise we'll capture the default amount of logs if the user explicitly says --accounting-logs/--daemon-logs=0.